.github/workflows/run-test-harness.yml

```diff
@@ -32,3 +32,4 @@ jobs:
           trigger_workflow: true
           wait_workflow: true
           client_payload: '{ "sha": "${{ github.event.pull_request.head.sha }}", "sdk": "nodejs" }'
+          comment_downstream_url: ${{ github.event.pull_request.comments_url }}
```

Comment:
is there anything else we want it to comment?

Reply:
let's land this first and see how it looks I guess